ambry-server/src/main/java/com.github.ambry.server/AmbryRequests.java

```diff
@@ -277,6 +277,9 @@ public void handleGetRequest(Request request) throws IOException, InterruptedExc
       responseSendTime = metrics.getBlobAllSendTimeInMs;
       responseTotalTime = metrics.getBlobAllTotalTimeInMs;
     }
+    if (getRequest.getClientId().startsWith(GetRequest.Replication_Client_Id_Prefix)) {
+      metrics.replicationGetBlobRequestRate.mark();
```

Comment:
`> 1` looks OK to me (but it still loses some accuracy since replication can also request one.)
The benefit is don't need to do string compare.

Size ranges? Replication and Get are different?